src/Rokt-Kit.js

```diff
@@ -243,6 +245,7 @@ var constructor = function () {
                 window.mParticle.Rokt.attachKit(self);
 
                 self.isInitialized = true;
+                console.warn('Rokt TRACE: launcher called');
```

Comment:
nit: launcher isn't relaly being called here, it's more like Rokt is now ready, right?

Reply:
You can ignore that. It was a debug log I forgot to remove.